src/sec_edgar_bulker/core/__init__.py

```
@@ -0,0 +1,45 @@
+from .idx_downloader import IdxDownloader
+from .header_generator import HeaderGenerator
+from .proxy_manager import ProxyManager
+from .models import (
+    Config, 
+    HeaderGeneratorSettings,
+    StaticHeaders,
+    ProxySettings,
+    DownloadSettings,
+    LoggingSettings,
+    DirectorySettings,
+    FilteringSettings
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)